### PR TITLE
initialize project using web-testexec-navigator as template

### PR DESCRIPTION
**Pull request by jjelschen**
Initial project setup analogous to web-testexec-navigator. Mostly just copy-pasted, after the generative approach using the latest Angular CLI caused too much hassle due to breaking changes in Angular 6.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        'source maps have to be turned off for ng cli in face of karma tests',
+        'for legible error messages in some cases (e.g. html template access missing function)',
+        'this configuration however is only used when running `karma start`'
+      ]
+    },
+    reporters: ['progress', 'kjhtml', 'coverage-istanbul'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    files: [ 'src/**/*.ts', 'src/**/*.css', 'src/**/*.html' ],
```
Comment (gunther-bachmann) [inline karma.conf.js]: possibly exclude node_modules?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@testeditor/testexec-details",
+  "version": "0.1.0",
+  "description": "Angular component for the navigation of test execution call hierarchies.",
```
Comment (gunther-bachmann) [inline package.json]: maybe: "Angular component providing a detailed view on test execution steps within test execution call hierarchies"

File: src/app/modules/details/test-exec-details.module.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TestExecDetailsComponent } from './test-exec-details.component';
+
+@NgModule({
+  imports: [
+    CommonModule
```
Comment (gunther-bachmann) [inline src/app/modules/details/test-exec-details.module.ts]: given that we use the MessagingService you could also add "MessagingService.forRoot()", right?

File: karma.conf.js (hunk 1 above)
Comment (jjelschen) [reply karma.conf.js]: `exclude: [ 'node_modules/**' ]` seems to have the effect of Karma not being able to access its own plugins… But there's `watchOptions.ignored`, which seems to do what we want.

File: src/app/modules/details/test-exec-details.module.ts (hunk 3 above)
Comment (jjelschen) [reply src/app/modules/details/test-exec-details.module.ts]: right, but I put it into `AppModule` instead, analogous to test-exec-navigator.

**Pull request merged**